public/locales/auset-fr.json

```diff
@@ -0,0 +1,426 @@
+      "setupTitle": "Configurer les Gardiens en fer",
+      "setup": "<i>1.</i> Mélangez les 12 <strong>jetons reliques</strong> face cachée. Placez-en un face cachée au hasard dans chaque forêt.<br/><i>2.</i> Choisissez-en 2 adjacents les clairières de la patrie sur le bord de la carte qui ont plus de 2 clairières entre elles et les patries ennemies. Mettez 4 <strong>guerriers</strong> dans chaque patrie.<br/><i>3.</i> Mettez les <strong> restants reliques</strong>, aussi uniformément que possible, parmi toutes les forêts qui ne sont pas adjacentes à vos terres natales.<br/><i>4.</i> Insérez une carte <strong>Fidèle Serviteur</strong> dans chacune des vos emplacements de colonne Suite."
+    },
+    "marquise": {
+      "name": "Marquise de Cat",
+      "setupTitle": "Configurer la Marquise de Cat",
+      "summaryTitle": "Joueur la Marquise",
+      "summary": "<p>En tant que marquise de Cat, vous voulez transformer la forêt en une centrale industrielle et militaire. Chaque fois que vous placez un <strong>bâtiment</strong> sur la carte, vous marquez des points. Plus il y a de ce type de bâtiment il y a sur la carte, plus vous marquez de points.</p><p>Pour alimenter cette construction, vous devez développer et protéger une économie interconnectée de <strong>bois</strong>. Construire des infrastructures rend vos virages plus efficaces et vous aide à piocher plus de cartes, alors frappez pour garantir votre droit à l'expansion. Votre armée est légion, vous permettant d'appliquer votre règle d'une poigne de fer, si nécessaire.</p><p>Le siège de votre pouvoir est le < strong>Donjon de la marquise de Cat</strong>, une structure si imposante qu'aucune autre faction ne peut placer de pièces dans sa clairière. De plus, vos <strong>Hôpitaux de campagne</strong> vous aideront à rester dans le combat. de vos guerriers sont retirés, vous pouvez dépenser une carte correspondant à la clairière des guerriers pour les replacer dans votre donjon - tant qu'il est en place. Protégez bien !</p>",
```

Comment:
Please remove the extra spaces in the diamond tags, this is causing errors in the rendering of this text.